modules/openapi-generator/src/main/resources/JavaSpring/allowableValues.mustache

```diff
@@ -0,0 +1 @@
+{{#allowableValues}}allowableValues = "{{#values}}{{{.}}}{{^-last}}, {{/-last}}{{#-last}}{{/-last}}{{/values}}"{{/allowableValues}}
```

Comment:
Does this handle enums and containers properly? I don't see a change to "values.mustache" but maybe that's already correct

Reply:
There is no values.mustache. Which file do you refer to?

Reply:
I was making an assumption about the reference to #values - does this work properly with enum values and containers then?